### ISSUE TITLE
Cannot manually choose "Mr Mime" or "Ho Oh"

### ISSUE THREAD

**Issue by canavan** [farkam135/GoIV#1025]
Since in Android 4.4, calibration is broken, I always have to enter the Pokemon species manually, the default is usually Alakazam. Entering and choosing "Mr. Mime" or "Ho Oh" from the drop down list however does not work properly, since GoIV just falls back to Alakazam after clicking "CHECK IV"..

**Comment by mokahless** [comment]
Ho-oh is stored as "Ho oh" but It doesn't come up for me when typing in "Ho" but does come up in the list when typing "H" or "Ho " (that's Ho followed by a space).

As for Mr. Mime, it seems to be stored as "Mr, Mime" (that's a comma) instead of "Mr. Mime" so try just typing "Mr" or "Mr," (that's a comma) then selecting from the list.

**Comment by canavan** [comment]
I can find those Pokemon in the drop down list and select them. Their name is then transferred to the text box, however, when I tap "Calculate IV", GoIV reverts to the mon that it erroneously chose before (usually Alakazam, sometimes Mew).

**Comment by mokahless** [comment]
I just experienced the issue you are describing. Just now on 5.2.8, I was attempting to evaluate a pokemon I found stats online for but it kept force-reverting to the automatically selected mon after I hit check IV. It only worked after I found another pokemon that was the same species and used that as a base.

As an aside, "Mr. Mime" is still "Mr, Mime" in 5.2.8. Not sure if this done on purpose for coding reasons or is a typo.
